docs/development/plugn-triggers.md

```diff
@@ -1,15 +1,15 @@
-# Pluginhooks
+# Plugn triggers
```

Comment:
Makes sense. Let's keep the old filename then and I'll include a note about pluginhooks moving to plugn triggers.